.kokoro/presubmit/securitycenter-it.cfg

```diff
@@ -0,0 +1,27 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
This file is to configure running integration tests found in the google-cloud-clients directory. Is that the intent here, or is this just for samples?

Reply:
I was intending this to be for samples, is there a mechanism to do that, or is it just best effort?